examples/adjoints/adjoints_ex6/femparameters.h

```diff
@@ -0,0 +1,149 @@
+
+#ifndef __fem_parameters_h__
+#define __fem_parameters_h__
+
+#include "libmesh/libmesh_common.h"
+#include "libmesh/dof_map.h"
+#include "libmesh/enum_norm_type.h"
+#include "libmesh/function_base.h"
+#include "libmesh/getpot.h"
+#include "libmesh/id_types.h"
+#include "libmesh/parallel_object.h"
+#include "libmesh/periodic_boundaries.h"
+#include "libmesh/periodic_boundary.h"
+
+#include <limits>
+#include <map>
+#include <string>
+#include <vector>
+
+
+
+class FEMParameters : public libMesh::ParallelObject
```

Comment:
> Yeah: e.g. take a look at adjoints_ex2/Makefile.am

Looks like that was trying to use the build system to set up symbolic links to headers in other examples, but what if the headers were themselves simply symbolic links to other files in the repo. Presumably the build system would work the same as always and not even know it was working with symbolic links?